project/R/conceptSets.R

```diff
@@ -78,6 +78,113 @@ conceptSets <- list(
+      4117045, # source: Arrhythmia therapy (procedures): Cardioversion, Rapid atrial pacing, temporary pacing, permanent pacing /// concept: Atrial overdrive pacing
+      4049398, # source: Arrhythmia therapy (procedures): Cardioversion, Rapid atrial pacing, temporary pacing, permanent pacing /// concept: Temporary cardiac pacemaker procedure
+      4051940, # source: Arrhythmia therapy (procedures): Cardioversion, Rapid atrial pacing, temporary pacing, permanent pacing /// concept: Permanent cardiac pacemaker procedure
+      609312 # source: Listed for heart transplantation /// concept: Awaiting transplantation of heart
+      ),
+    
+    UC3PateintCharacteristics = c(
+      434007, # source: hemophilia A diagnosis /// concept: hereditary factor VIII deficiency disease
+      4094223, # source: hemophilia A subtype/ severity /// concept: mild hereditary factor VIII deficiency disease
+      4140661, # source: hemophilia A subtype/ severity /// concept: moderate hereditary factor VIII deficiency disease
+      4056830, # source: hemophilia A subtype/ severity /// concept: severe hereditary factor VIII deficiency disease
+      3003694, # source: blood group /// concept: ABO and Rh group [Type] in Blood
+      37393608, # source: factor VIII inhibitor status /// concept: factor VIII inhibitor activity
+      4126681, # source: factor VIII inhibitor status /// concept: detected
+      9190 # source: factor VIII inhibitor status /// concept: not detected
+    ),
+    
+    UC3LabMeasurements = c(
+      3024942, # source: factor VIII inhibitor titer /// concept: coagulation factor VIII inhibitor [Units/volume] in platelet poor plasma by coagulation assay
+      3022520, # source: factor VIII activity measurement /// concept: coagulation factor VIII activated [Units/volume] in platelet poor plasma by coagulation assay
+      3011832, # source: factor VIII activity measurement /// concept: coagulation factor VIII activity [Units/volume] in platelet poor plasma by Chromogenic assay
+      43534000, # source: Von Willebrand factor activity measurement /// concept: von Willebrand factor (vWf) activity [Units/volume] in platelet poor plasma by Immunoassay
+      3002124, # source: Von Willebrand factor antigen measurement /// concept: von Willebrand factor (vWf) Ag [Units/volume] in platelet poor plasma by Immunoassay
+      3023693, # source: Von Willebrand factor propeptide measurement /// concept: von Willebrand factor (vWf) multimers in platelet poor plasma by Immunoblot
+      3042349, # source: Von Willebrand factor propeptide measurement /// concept: von Willebrand factor (vWf) cleaving protease inhibitor [Units/volume] in platelet poor plasma
+      4175016, # source: activated partial thromboplastin time (APTT) /// concept: partial thromboplastin time, activated
+      3034426, # source: Prothrombin time (PT) /// concept: prothrombin time (PT)
+      37393863, # source: platelet count /// concept: platelet count
```

Comment:
This is a not standard concept. The standard one is: [Platelet count](https://athena.ohdsi.org/search-terms/terms/4267147)